index.js

```diff
@@ -17,10 +17,11 @@ function propagate(events, source, dest) {
 
   source.emit = function(eventType) {
```

Comment:
Since `emit` should return whether there were listeners or not, we should check both EventEmmiters and return true if we find one in either of them, right?

```js
source.emit = function(eventType) {
  var handled = oldEmit.apply(source, arguments);

  if (! events || ~events.indexOf(eventType)) {
    handled = dest.emit.apply(dest, arguments) || handled;
  }

  return handled;
}
```

Reply:
Agreed, thanks.